### ISSUE TITLE
Agrarian 2 Server Issues

### ISSUE THREAD

**Issue by redhairydevil** [jadedcat/AS2Configs#95]
trying to make server, gets passed witching gadgets the does this and i cant get past it.
it repeats this over and over. Pls halp i is stuck
![098abc76f1ce16c4ac000231d404d2a7](https://cloud.githubusercontent.com/assets/14058794/9584559/af4e3e38-5009-11e5-9754-6d9eb47acca8.png)
 
![225f3cda1b887b8f6c0ab5ea6c9d71a5](https://cloud.githubusercontent.com/assets/14058794/9584819/5124fe76-500b-11e5-8bfd-fb23c009b2aa.png)

I have tried using a batch file to load it and gave it memory but nothing ive tried seems to work :(
![727c39df7d2f73ddb0a1b82cf67afd89](https://cloud.githubusercontent.com/assets/14058794/9584601/f2f06d96-5009-11e5-85af-d1d40c9439af.png)

Using latest version (1.7.10, server 2-1.1.8)

**Comment by MarshMilo100** [comment]
you do not have enough memory allocated. I think this is due to you not propely writing the batch file, change '-xmx4g' to '-Xmx4G', and '-xms4g' to '-Xms4G'. this should fix your memory issues.

**Comment by MarshMilo100** [comment]
also try using the 1.1.13 server files.

**Comment by MarshMilo100** [comment]
btw this is my batch file to start the 1.1.13 server. if it helps. http://pastebin.com/ayGNj1q5

**Comment by redhairydevil** [comment]
ok thanks but still not working, tried the 1.1.13 files and it still said the same thing but i have tried the 1.1.8 (changing what you said and it is now saying a mod tried to load an existing class with this crash report

http://pastebin.com/YvzivsLk

**Comment by pjfranke** [comment]
I tested just now with a fresh download and Java8u60, and this command works on Windows:

java -Xms1024m -Xmx2048m -XX:PermSize=256m -jar forge-1.7.10-10.13.4.1469-1.7.10-universal.jar nogui

The "-XX:PermSize=256m" flag seems to be necessary on Java7, but as the error below notes, doesn't work in Java8.  It will not harm anything and can be ignored.

Java HotSpot(TM) 64-Bit Server VM warning: ignoring option PermSize=256m; support was removed in 8.0

2GB max allocated as it's a virtual machine with only 4GB of RAM.
